framework/assets/yii.validation.js

```diff
@@ -436,5 +436,32 @@ yii.validation = (function ($) {
         }
     }
 
+    function validateMimeType(mimeTypes, fileType)
+    {
+        for (var i=0; i<mimeTypes.length; i++) {
```

Comment:
Seems to be micro optimization as of access to `length` property complexity is O(1) and it's value stored in hashtable. If you still sure to replace with variable I do it